keyboards/handwired/dactyl_manuform/4x5/keymaps/kombo/config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "../../config.h"
+
+#ifdef MANUFACTURER
+#undef MANUFACTURER
+#endif
+#define MANUFACTURER domonkos
+
+#ifdef DESCRIPTION
+#undef DESCRIPTION
+#endif
+#define DESCRIPTION endgame
+
+#ifdef PRODUCT
+#undef PRODUCT
+#endif
+#define PRODUCT DacMan
+
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This is the default behavior now, and should be removed (as this doesn't do anything, actually).

```suggestion
```